app/controllers/question_controller.rb

```diff
@@ -9,10 +9,10 @@ class QuestionsController < ApplicationController
 	respond_to :html, :json
 
 	def index
-		@questions = Question.all
-	end	
+		@questions = Question.order("id DESC")
```

Comment:
For comments you would need to filter based upon the commentable, so yeah, this is probably not as straightforward.